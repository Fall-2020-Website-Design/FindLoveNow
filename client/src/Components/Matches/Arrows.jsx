import React, { Component } from 'react';
import Row from 'react-bootstrap/Row';
import Col from 'react-bootstrap/Col';
import Image from 'react-bootstrap/Image';
import arrow from '../../Images/arrow.svg';

export class Arrows extends Component {    
    render() {
        return (
            <div>
                <Row>
                    <Col className="d-flex justify-content-between arrows" >
                        <Image id="left" alt="left" src={arrow} className="align-top left-arrow" onClick={this.props.prev} fluid />
                        <Image id="right" alt="right" src={arrow} className="align-top right-arrow" onClick={this.props.next} fluid />
                    </Col>
                </Row>
            </div>
        )
    }
}

export default Arrows;