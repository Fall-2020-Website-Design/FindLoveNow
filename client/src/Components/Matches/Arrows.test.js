import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { Arrows } from './Arrows';


Enzyme.configure({ adapter: new Adapter() });

describe('Arrows', () => {

	let wrapper;

	beforeEach(() => {
		wrapper = shallow(<Arrows  />);
    });
    
    test('renders', () => {
		const component = renderer.create(
			<MemoryRouter>
				<Arrows  />
			</MemoryRouter>
		).toJSON();
		expect(component).toMatchSnapshot();
    });
    
    test('left arrow', () => {
		wrapper.find('#left').simulate('click');
		
    });
    
    test('right arrow', () => {
        wrapper.find('#right').simulate('click');
	});
});