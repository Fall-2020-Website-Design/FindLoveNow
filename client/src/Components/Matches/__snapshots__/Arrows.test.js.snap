// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Arrows renders 1`] = `
<div>
  <div
    className="row"
  >
    <div
      className="d-flex justify-content-between arrows col"
    >
      <img
        alt="left"
        className="align-top left-arrow img-fluid"
        id="left"
        src="arrow.svg"
      />
      <img
        alt="right"
        className="align-top right-arrow img-fluid"
        id="right"
        src="arrow.svg"
      />
    </div>
  </div>
</div>
`;
